### PR TITLE
Portfolio Desktop Version

### PR DESCRIPTION
**Pull request by Johnadibe**
- Used media queries to display desktop screens
- Built with CSS Grid and Flexbox
- Replicated the design as much as possible from the selected Figma Template

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -464,3 +491,991 @@ hr {
   justify-content: center;
   gap: 2.4rem;
 }
+
+@media screen and (min-width: 48em) and (max-width: 60em) {
```
Comment (vechicin) [inline style.css]: Please change all of your media queries to `px` as `em` is a relative measure, not a static one, and it is not a good way to set breaking points for different screen sizes. Additionally, make sure you only use media queries for breaking points starting at 768px following the principles of mobile-first development.

File: index.html [hunk 2]
```diff
@@ -69,125 +82,134 @@ <h1 id="headline-primary">
...
+            </div>
 
             <div class="about-img"></div>
         </div>
 
-        <hr />
+        <hr class="about" />
 
         <div class="about-categories">
             <div class="categories">
```
Comment (vechicin) [inline index.html:215]: The 2nd category should say `Frameworks`, not `Language` in order to match the technologies listed below. Please fix the header.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Johnadibe ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] README
- [x] Good communication with the reviewer
- [x] Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
# **Status: Approved** :tada: 

Hi @Johnadibe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Followed Github Flow. :heavy_check_mark: 
-  Meaningful commit messages. :heavy_check_mark: 
-  HTML && CSS Best Practices. :heavy_check_mark: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Johnadibe) [reply style.css]: Okay @vechicin. I will do that right a way.😊

File: index.html (hunk 2 above)
Comment (Johnadibe) [reply index.html:215]: Okay Thank you @vechicin.

**Comment by Johnadibe** [comment]
Thanks😊

**Pull request merged**